src/auth/auth.service.ts

```diff
@@ -29,6 +31,16 @@ export class AuthService {
     return this.jwtService.verifyAsync(jwtToken);
   }
 
+  async successfullLoginToken(id: number, email: string): Promise<string> {
```

Comment:
how about something like `generateUserBearerToken`? You could include strategy name in the method name but I don't remember the strategy names you have.